.gitignore

```diff
@@ -114,6 +114,7 @@ app.*.symbols
 !/dev/ci/**/Gemfile.lock
 
 # secrets storage
-secrets.json
```

Comment:
czemu out?

Reply:
nie mam pojęcia jak to się stało, poprawiam